src/js/components/Accordion.js

```diff
@@ -63,7 +70,12 @@ export default class Accordion extends Component {
 
 Accordion.propTypes = {
   animate: PropTypes.bool,
+  disabled: PropTypes.bool,
   openMulti: PropTypes.bool,
+  icon: PropTypes.oneOfType([
+    PropTypes.element,
+    PropTypes.bool
```

Comment:
These are the designs I've received for mobile view. The footer links on mobile view becomes an Accordion without an icon.

![image](https://cloud.githubusercontent.com/assets/3210082/17790906/54681f66-6534-11e6-9ad1-c24ff7b9f497.png)